src/main/groovy/io/openliberty/tools/gradle/tasks/GenerateFeaturesTask.groovy

```diff
@@ -224,6 +230,17 @@ class GenerateFeaturesTask extends AbstractFeatureTask {
         servUtil.setLowerCaseFeatures(true);
         return existingFeatures;
     }
+
+    // returns the features specified in the generated-features.xml file
+    private Set<String> getGeneratedFeatures(ServerFeatureUtil servUtil, File generatedFeaturesFile) {
+        servUtil.setLowerCaseFeatures(false);
+        Set<String> genFeatSet = new HashSet<String>();
+        servUtil.getServerXmlFeatures(genFeatSet, configDirectory,
```

Comment:
```suggestion
        servUtil.getServerXmlFeatures(genFeatSet, server.configDirectory,
```

Reply:
As a future item, we could refactor this logic to BinaryScannerUtil or another util class in ci.common as it is very similar across ci.maven & ci.gradle. We have similar logic (but not identical) in DevUtil: https://github.com/OpenLiberty/ci.common/blob/4b19329fd1475ce925979f37144f9f5041062aaf/src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java#L5589-L5596